misc/scripts/build.sh

```diff
@@ -58,14 +58,14 @@ then
    exit 2
 fi
 
-min_ver=`echo $ver | grep -o '\.[0-9]*\.' | sed 's/\.//g'`
-if [ $min_ver -lt 8 ]
+maj_ver=`echo $ver | sed 's/\..*$//'`
+min_ver=`echo $ver | sed 's/[0-9]*\.//' | sed 's/\..*$//'`
+if [ $maj_ver -lt 17 -a $min_ver -lt 8 ]
```

Comment:
Hmm Is this condition correct? maj_ver < 17 && min_ver < 8? Why?

I think we can get rid of this code altogether and just document.